### PR TITLE
print Job with uppercase J when logging that a runonce job ran already

### PR DESCRIPTION
**Pull request by castilma**
In other log messages, Job is written with uppercase j.
Adjust this log lines to print uppercase j, too.

### CODE REVIEW

File: database.c [hunk 1]
```diff
@@ -1453,7 +1453,7 @@ set_next_exe_startup(struct cl_t *cl, const int context,
     if (is_runonce(cl->cl_option) && is_hasrun(cl->cl_option)) {
         /* if we get here, then context != CONTEXT_BOOT and_volatile is false */
         /* do nothing: don't re-schedule or add to the job queue */
-        explain("job '%s' with runonce set has already run since last "
+        explain("Job '%s' with runonce set has already run since last "
```
Comment (yo8192) [inline database.c:1456]: why not, but  a `git grep '"job' *.c *.h'` finds a number of other examples where 'job' has a lower-case 'j' at the beginning of a message/sentence. If we are doing this, you should do all of them too.

**Review by yo8192 — CHANGES_REQUESTED** [review]

File: conf.c [hunk 2]
```diff
@@ -294,7 +294,7 @@ synchronize_file(char *file_name, int is_system_startup)
                             && is_runonce(old_l->cl_option)
                             && is_hasrun(old_l->cl_option)) {
                             explain
-                                ("  from last conf: job '%s' with runonce set has "
+                                ("  from last conf: Job '%s' with runonce set has "
```
[the remark this answers is not in the document]
Comment (castilma) [reply conf.c]: understandable.

File: database.c (hunk 1 above)
Comment (castilma) [reply database.c:1456]: did it. Are two commits ok or should they be one?

**Review by yo8192 — APPROVED** [review]

**Comment by yo8192** [comment]
2 commits is just fine -- I squashed them into one when merging.
Thanks!

**Pull request merged**